week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -32,17 +32,23 @@ var parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
+  return house.currentOwner.firstName + " " + house.currentOwner.lastName;
 
 }
 
 // returns an array of the owners' email addresses of the two houses
 function getEmailAddresses(house1, house2) {
-
+  return house1.currentOwner.email + " ," + house2.currentOwner.email;

```

Comment:
Please double check the instructions. There function doesn't return the correct type at the moment. Can you fix it?